### PR TITLE
fix: fix markdown link

### PR DESCRIPTION
**Pull request by josmithua**
Fix markdown link in blog post (https://stately.ai/blog/state-management-how-to-tell-a-bad-boolean-from-a-good-boolean)

### CODE REVIEW

**Review by davidkpiano — APPROVED** [review]

**Comment by josmithua** [comment]
One other comment:

One line 116, the post says: 

> A couple of folks in the comments are asking how we...

A reflection that this post was copied from another platform. Maybe we should reword this?

**Review by laurakalbag — APPROVED** [review]

**Comment by laurakalbag** [comment]
> One other comment:
> 
> One line 116, the post says:
> 
> > A couple of folks in the comments are asking how we represent a state enum in Javascript. While the above code will work without typings, you can also represent enums as an object type.
> 
> A reflection that this post was copied from another platform. Maybe we should reword this to something like:
> 
> > We can represent a state enum in Javascript as well. While the above code will work without typings, you can represent enums as an object type.

Great suggestion, thanks!
